x-pack/test/search_sessions_integration/tests/apps/dashboard/async_search/save_search_session.ts

```diff
@@ -72,7 +78,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       ).to.be(fakeSessionId);
     });
 
-    it('Saves and restores a session', async () => {
+    it('Saves and restores a session - miaou', async () => {
```

Comment:
is this a cat test ? :)

Reply:
Oh I forgot it! 😂 Thanx I will remove it :D